test/test_new_world_grudge.py

```diff
@@ -0,0 +1,367 @@
+__copyright__ = """
+Copyright (C) 2021 University of Illinois Board of Trustees
```

Comment:
I'm guessing you're already doing this, but I'd like it agreed upon as a policy: What I'd like here to make reviewing easier is to have a file where there's a 1-to-1 correspondence between new-world and old-world tests, to make sure we don't lose coverage.